app/src/main/java/de/voicegym/voicegym/recordActivity/views/util/ScalingFunction.kt

```diff
@@ -0,0 +1,45 @@
+package de.voicegym.voicegym.recordActivity.views.util
+
+import kotlin.math.roundToInt
+
+data class PixelFrequencyPair(val pixelPosition: Int, val correspondingFrequency: Double)
+
+abstract class ScalingFunction(val from: PixelFrequencyPair, val until: PixelFrequencyPair) {
```

Comment:
why is it an abstract class and not an interface?

Reply:
because i intend to store from and until